src/sail_lean_backend/pretty_print_lean.ml

```diff
@@ -76,6 +76,7 @@ let rec doc_typ (Typ_aux (t, _) as typ) =
   | Typ_id (Id_aux (Id "int", _)) -> string "Int"
   | Typ_id (Id_aux (Id "bool", _)) -> string "Bool"
   | Typ_id (Id_aux (Id "bit", _)) -> parens (string "BitVec 1")
+  | Typ_id (Id_aux (Id "nat", _)) -> string "Nat"
```

Comment:
Can you use a `Nat` type where an `Int` is expected in Lean?  Sail treats nats as a form of integer.

Reply:
Yes, `Nat`s are coerced automatically to `Int` by Lean where needed